SmallCommitsWorkshop/Controllers/UsersController.cs

```diff
@@ -16,8 +16,8 @@ public class UsersController : Controller {
 			m_usersContext = usersContext;
 
 			if( !m_usersContext.Users.Any() ) {
-				m_usersContext.Users.Add( new User() { Id = 169, UserName = "D2LSupport" } );
-				m_usersContext.Users.Add( new User() { Id = 175, UserName = "user1" } );
+				m_usersContext.Users.Add( new User() { Id = 169, UserName = "D2LSupport", IsActive = true } );

```

Comment:
`IsActive` isn't used anywhere in "real" code.  Is this on purpose?

Reply:
It is not on purpose. It is passed back to the client when they make requests, but otherwise is not used.

Reply:
I think we can work with that.